worker/src/test/java/gov/cms/ab2d/worker/processor/JobDataWriterTest.java

```diff
@@ -15,11 +15,12 @@
 
 class JobDataWriterTest {
```

Comment:
I wonder if this test should be renamed to `TextStreamHelperTest`, since the production class is no longer named `JobDataWriterImpl`

Reply:
I've created a new TextStreamHelperImplTest. This was just to make sure I didn't break any previous expectations. More regression testing. We could name it something else??

Reply:
okay. Did not realize that. Thanks.